terraform/variables.tf

```
@@ -0,0 +1,34 @@
+variable "app_name" {
+  description = "Application name"
+  type        = string
+  default     = "mlmd"
+}
+
+variable "channel" {
+  description = "Charm channel"
+  type        = string
+  default     = null
+}
+
+variable "config" {
+  description = "Map of charm configuration options"
+  type        = map(string)
+  default     = {}
+}
+
+variable "model_name" {
+  description = "Model name"
+  type        = string
+}
+
+variable "resources" {
+  description = "Map of resources revisions"
```

This is not a resource revision
```suggestion
  description = "Map of resources"
```